http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -188,24 +196,27 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         AppCreationException(s"Pet not found for user ${app.auditInfo.creator}", Some(ctx.traceId))
       )
 
-      // If we're configured to use pod identity with the pet for this app, resolve pet managed identity in Azure
-      // and assign the VM scale set.
-      // See https://broadworkbench.atlassian.net/browse/IA-3804 for tracking migration to AKS Workload Identity
-      // for all app types.
-      petMi <- identityType match {
-        case PodIdentity =>
+      // private apps run as the pet managed identity, get it
```

Comment:
I don't know why I read this comment with an attitude 😂 

"Private apps run as the pet managed identity, capisce?"